Added tooltip property for sidenav menu items.

servoystuff opened the pull request:
This change adds the ability to set a "tooltip" property in sidenav menu items. Pretty straightforward but needed by one of my client... Thanks for merging!

seanthomasdevlin commented:
Support case created. Please subscribe to the support ticket for updates:
https://support.servoy.com/browse/SVYX-432

jcompagner commented:
we also now need an Titanium NGClient impl for these kind of patches..

servoystuff commented:
Sorry but I've added what I thought was needed to the ng2 implementation, mainly the markup in sidenav.html and added "tooltip" property in sidenav.ts, based on what I could see for equivalent navbar implementation, but it looks like there's still something missing in sidenav.ts and I have no idea what. Please take care of that part, thanks!
